apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1099,6 +1099,13 @@
                     "maximum" : 32,
                     "default" : 32
                 },
+                "ping_connected_peer_percentage" : {
```

Comment:
I'm wondering if there should be a sub-group for ping messages? Wouldn't it, for example, also be reasonable to be able to set an upper limit on the actual number of peers included?